.ignore

```diff
@@ -0,0 +1 @@
+fixtures/
```

Comment:
`rg` and `ag` respect application-agnostic `.ignore` files, this one says not to search the contents of any subdirectory named `fixtures`.  Those are files that should be in git but they make searching the repo annoying, since they contain almost any string that I wanted to look for.